File: app/src/main/java/com/egecius/demo_androidplatform/fragments/Fragment3.java
package com.egecius.demo_androidplatform.fragments;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.egecius.demo_androidplatform.R;

public class Fragment3 extends Fragment {

    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
            @Nullable Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment3, container, false);
        view.findViewById(R.id.pop_fragment_2).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                popTransactionFragment2();
            }
        });
        return view;
    }

    private void popTransactionFragment2() {
        ((MyFragmentActivity) getActivity()).popTransactionFragment2();
    }
}

File: app/src/main/java/com/egecius/demo_androidplatform/fragments/Fragment3.kt
package com.egecius.demo_androidplatform.fragments

import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.egecius.demo_androidplatform.R

class Fragment3 : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.fragment3, container, false)
        view.findViewById<View>(R.id.pop_fragment_2).setOnClickListener { popTransactionFragment2() }
        return view
    }

    private fun popTransactionFragment2() {
        (activity as MyFragmentActivity).popTransactionFragment2()
    }
}
